recon_surf/recon-surf.sh

```diff
@@ -753,7 +744,7 @@ else
     RunIt "$cmd" $LF $CMDF
     
     # for hires decimate mesh 
-    if [ "$vox_size" == "auto" ]; then
+    if [ "$vox_size" -lt "$hires_voxsize_threshold" ]; then
```

Comment:
just check if hires is set, no need for the -lt.

Reply:
Personally, I would clear statements and `[ ! -z "$hiresflag" ]` does not really seem "clearer" to me, i.e. I know have to look up what empty `$hiresflag` means...

Reply:
I think hiresflag is very descriptive = hires processing whereas voxsize and voxsize_threshold is unclear.